pkgs/applications/audio/guitarix/default.nix

```diff
@@ -22,15 +23,14 @@ stdenv.mkDerivation rec {
   buildInputs = [
     avahi bluez boost eigen fftw glib glibmm gtk gtkmm jack2
     ladspaH librdf libsndfile lilv lv2 serd sord sratom
+    zita-convolver zita-resampler
   ];
 
   configureFlags = [
     "--shared-lib"
     "--no-desktop-update"
-    "--no-faust" # Need to package a release of faust, 0.9.58 or 0.9.65
     "--enable-nls"
-    "--includeresampler" # Zita-resampler not packaged, use vendored version
-    "--includeconvolver" # Zita-convolver not packaged, use vendored version
+    "--no-faust" # todo: find out why --faust doesn't work
```

Comment:
@codyopel 
The reason the current faust pkg uses the git version, is because the latest release has a serious bug (doesn't work with OSC), and there hasn't been a release in a while.

Other than that, I'm not sure what you mean.  You did see I changed the last line of my previous remark from "will test" to "have tested"?